### DISCUSSION TITLE
有两个TripleDES解密，没有TripleDES加密

### DISCUSSION THREAD

**Discussion by baspatos** [he3-app/he3-feedback#320] [Bug Report]
### Version
tripledes-decryption 1.0.4

### Environment
Windows v2.0.15

### What is actually happening?
1.有两个TripleDES解密，没有TripleDES加密。
2.其中一个应该是加密，其中一个应该是解密；加密的里面缺少PKCS5的填充方式，输出缺少选择使用hex输出或base64输出。

**Comment by zhenweiwang1990** [comment]
修复了，打开这个： https://t.he3app.com/t22y

**Comment by zhenweiwang1990** [comment]
此外我们已经开源了这两个工具，如果你有额外需求，欢迎贡献： https://github.com/he3-app/tools-example/tree/main/triple-des-encryption
